lib/components/settings/widgets/settings_card.dart

```diff
@@ -110,11 +110,28 @@ class SettingsCard extends SettingsElementModel {
 
 class _SettingsCardState extends State<SettingsCard> {
   late bool _value;
+
+  late bool isMutable;
```

Comment:
would be better to have those as getters cuz widget.type could change anytime and it could mess up stuff